dbt_airflow_factory/tasks_builder/builder.py

```diff
@@ -162,24 +164,16 @@ def _create_tasks_from_graph(self, dbt_airflow_graph: DbtAirflowGraph) -> ModelE
 
     def _make_dbt_tasks(self, manifest_path: str) -> ModelExecutionTasks:
         manifest = self._load_dbt_manifest(manifest_path)
-        dbt_airflow_graph = self._create_tasks_graph(manifest)
+        dbt_airflow_graph: DbtManifestGraph = create_tasks_graph(
+            self.gateway_config,
+            manifest,
+            self.airflow_config.enable_dags_dependencies,
+            self.airflow_config.show_ephemeral_models,
```

Comment:
Wouldn't be better to wrap the configuration into a config class? Open-close principle - for libs very important.

Reply:
This probably needs to be changed in the lib in the first place. But we can do that.

Reply:
yes please

Reply:
Done.